Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -234,33 +236,41 @@ HTMLLinkElement::LinkProcessingOptions HTMLLinkElement::create_link_options()
 JS::GCPtr<Fetch::Infrastructure::Request> HTMLLinkElement::create_link_request(HTMLLinkElement::LinkProcessingOptions const& options)
 {
     // 1. Assert: options's href is not the empty string.
+    VERIFY(options.href != "");
```

Comment:
We generally use `is_empty()` for these kinds of checks, so `VERIFY(!options.href.is_empty())` would be better here.

Reply:
I'll make this change :)